streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SavePointServiceImpl.java

```diff
@@ -114,6 +127,41 @@ public SavePoint getLatest(Long id) {
     return this.getOne(queryWrapper);
   }
 
+  @Override
+  public void trigger(Long appId, @Nullable String savepointPath) {
+    log.info("Start to trigger savepoint for app {}", appId);
+    Application app = appService.getById(appId);
+    AssertUtils.notNull(app, String.format("The application %s doesn't exist.", appId));
+    String trackingUrl = app.getJobManagerUrl();
+    AssertUtils.state(
+        StringUtils.isNotEmpty(trackingUrl),
+        String.format("The flink trackingUrl for app[%s] isn't available.", appId));
+    String jobId = app.getJobId();
+    AssertUtils.state(
+        StringUtils.isNotEmpty(jobId),
+        String.format("The jobId of application[%s] is absent.", appId));
+    String triggerId = triggerSavepoint(trackingUrl, jobId, savepointPath);
+    log.info("Request savepoint successful in triggerId {}", triggerId);
+  }
+
+  private String triggerSavepoint(
+      String trackingUrl, String jobId, @Nullable String savepointPath) {
+    String response = null;
+    try {
+      String url = String.format("%s/jobs/%s/savepoints", trackingUrl, jobId);
+      String postBody =
+          OBJECT_MAPPER.writeValueAsString(new SavepointTriggerRequestBody(savepointPath, false));
+      response = HttpClientUtils.httpPostRequest(url, postBody);
+      TriggerResponse triggerResponse = OBJECT_MAPPER.readValue(response, TriggerResponse.class);
+      return triggerResponse.getTriggerId().toString();
+    } catch (Exception e) {
+      throw new RuntimeException(
```

Comment:
Hi @wolfboys, we should use the `ApiAlertException` here, right?

Reply:
> Hi @wolfboys, we should use the `ApiAlertException` here, right?

Yes,  we should use the `ApiAlertException`, savepoint manually triggered by the user . if an exception occurs,  should be exposed to the user

Reply:
I see @RocMarshal  changed it to `ApiException`, @wolfboys , could you please help check again? Should we use the `ApiAlertException` or `ApiException`?

Reply:
@1996fanrui @wolfboys Should it be `ApiAlertException` ?
But the current coding is `AlertException` instead of `ApiAlertException`.

Reply:
`AlertException`:  notify users of some info by WeChat,DingTalk, Msg..., 
`ApiAlertException`:  some message will be notify to user through the front-end pop-up window of streampark,
so, we should use the `ApiAlertException` here, not `AlertException `

Reply:
> I see @RocMarshal changed it to `ApiException`, @wolfboys , could you please help check again? Should we use the `ApiAlertException` or `ApiException`?

`ApiAlertException `